salt/journal/config/srv-journal-app-config-parameters.yml

```diff
@@ -1,5 +1,16 @@
+{% set on_elasticache = salt['elife.cfg']('cfn.outputs.ElastiCacheHost') %}
+{% set standard_keys = ['secret', 'api_key', 'api_url', 'api_url_public', 'side_by_side_view_url', 'session_name', 'status_checks', 'gtm_id', 'disqus_domain'] %}
+
 parameters:
-    {% for name in ['secret', 'api_key', 'api_url', 'api_url_public', 'side_by_side_view_url', 'session_name', 'status_checks', 'gtm_id', 'disqus_domain', 'redis_cache', 'redis_http_cache'] %}
+    {% if not on_elasticache %}
+    {% do standard_keys.extend(['redis_cache', 'redis_http_cache']) %}
+    {% else %}
+    {% set remote_redis = "redis://" + salt['elife.cfg']('cfn.outputs.ElastiCacheHost') + ":" + salt['elife.cfg']('cfn.outputs.ElastiCachePort') %}
+    redis_cache: {{ remote_redis }}
+    redis_http_cache: {{ remote_redis }}
```

Comment:
Worth installing a new extension if the local cache is not used much?
There are only two configuration keys at the moment however, how should I configure them? Since https://github.com/elifesciences/builder-base-formula/pull/44 is not merged the local Redis is preserved at the moment, and can still be used.